test/binary-op.civet

```diff
@@ -146,3 +146,35 @@ describe "binary operations", ->
     ---
     a instanceof b
   """
+
+  testCase """
+    pipe expression
+    ---
+    a |> fn
+    ---
+    (fn)(a)
+  """
+
+  testCase """
+    chained pipe expression
+    ---
+    a |> foo |> bar
+    ---
+    (bar)((foo)(a))
```

Comment:
Similarly here should ideally be `bar(foo(a))`, only wrapping with parens if the expression is more complex.